PROTOCOL.md

```diff
@@ -424,6 +429,67 @@ enough to get you started. You can watch the data sent and received from
 the server on a regular connection, or look at the client source code
 for a full list of message types.
 
+**Action requests**
+
+These are how the client sends the player's decisions to the server. Most
+requests are sent with `REQUESTID`. `REQUESTID` is used to identify which
+action the request was intended for and is used to protect against race
+conditions with the cancel button. The first decision of a match has a
+`REQUESTID` of 1, and each successful decision increments the counter by 1.
+`REQUESTID` is optional.
```

Comment:
I'd say that all of the following commands can have `|REQUESTID` added to the end of them, but not otherwise mention `|REQUESTID` in each of the individual sections.